src/Bridges/Constraint/bridges/slack.jl

```diff
@@ -46,6 +46,26 @@ function MOI.get(
     return [bridge.slack_in_set]
 end
 
+function MOI.get(
+    ::_AbstractSlackBridge{T,VF,S,F,S},
```

Comment:
This case can't happen in reality, because it'd mean that the added set was the same type as the original set. So we can just error.